### DISCUSSION TITLE
v0.45.0-20231122.130444

### DISCUSSION THREAD

**Discussion by qvacua** [qvacua/vimr#1019] [General]
* VimR is not using custom build binary based on `libnvim` anymore. Thanks to GH-1015 by [georgeharker](https://github.com/georgeharker), VimR now uses stock Neovim and can even use external Neovim binary (from v0.10.0 on).
* Neovim 0.10.0-dev which means Neovim itself may have some rough edges.
* Minimum macOS requirement is now Ventura (v13).
* Fix VimRSetCharacterspacing: GH-1014
* Add basic file handling to the filebrowser: GH-1005
* Provide neovim original commandline interface: GH-1003
* Add an option for custom Markdown processor: GH-745
* Dependencies updates:
  - ReactiveX/RxSwift@6.6.0
  - sparkle-project/Sparkle@2.5.0
  - apple/swift-collections@1.0.5
  - Quick/Nimble@13.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/qvacua/vimr/releases/tag/v0.45.0-20231122.130444'>v0.45.0-20231122.130444</a>.</em>

**Comment by metalelf0** [comment]
Hi, I downloaded the latest nightly but I'm getting the error "is damaged and can’t be opened. You should move it to the Trash". Tried `xattr -cr` on the app file but opening it shows a stacktrace (let me know if you need it). I'm on an M1 Macbook Pro.

**Reply by nmder** [reply]
Perhaps try `xattr -d com.apple.quarantine VimR.app` to remove it from the quarantine

**Reply by metalelf0** [reply]
It's saying it's not quarantined:  `xattr: VimR.app: No such xattr: com.apple.quarantine`.

**Reply by nmder** [reply]
Perhaps try this:
``` bash
spctl --add --label "VimR" VimR.app
spctl --enable --label "VimR"
```

**Reply by metalelf0** [reply]
It's doing nothing. The app starts, but it still opens the same stacktrace and force closes.

**Reply by nmder** [reply]
Hmm. Did you try the latest nightly that was built few hours ago? It's working on mine (Intel).

**Reply by metalelf0** [reply]
I just tried and I have the same error. App is damaged. Maybe there's something wrong with M1 macs?

**Reply by georgeharker** [reply]
I’ll download the official release and take a look today.

**Reply by georgeharker** [reply]
The current build downloads and works fine for me (i'm on M1) - what OS version are you on?

**Comment by metalelf0** [comment]
I'm on Sonoma 14.1.1.

**Reply by georgeharker** [reply]
Oh same here. Would you be able to find the crash report in console and post it?

**Comment by metalelf0** [comment]
Sure. Here it is (for reference, this is after `xattr -cr
/Applications/VimR.app`): https://gist.github.com/metalelf0/5257dea8a95bdce8a1568aaf8769d7ce

**Reply by georgeharker** [reply]
In system prefs for security do you see any marker indicating the option to trust the app?

I’m pretty confused as to why this is occurring. Which security setting for trusted developers etc do you have set?

**Comment by technoch1ef** [comment]
Hello. I recently updated to VimR 0.45 and after update `FZF` and `vim-esearch` works really slow for me. I'm working on very large project, but I did not have any issues with it prior. Now it hangs for solid 10 seconds when I open `fzf`, and  `vim-esearch` hangs vim

this was not an issue before the update. I also tried to do `:PlugUpdate` but that didn't help

**Reply by technoch1ef** [reply]
I was forced to downgrade. Version 0.44 works wonders without any lag and this was getting unbearable

**Reply by georgeharker** [reply]
Have you tried with an external / different version of nvim? Does this only occur with a specific version of fzf?

**Reply by technoch1ef** [reply]
No, I haven't tried another versions of nvim
I'm currently using NVIM v0.9.1

As for the the version of fzf or vim-esearch, I did run `:PlugUpdate` when trying to fix the problem but that didn't help, so both plugins should be the latest version

I will try updating both nvim and vimr and will report back

**Reply by technoch1ef** [reply]
Sorry, I haven't had time to check other versions of nvim, but I've just installed the latest VimR release and it works like a charm. This is great! I love it  🎉
